vllm/core/block/cpu_gpu_block_allocator.py

```diff
@@ -149,6 +162,8 @@ def free(self, block: Block) -> None:
         Args:
             block (Block): The block to be freed.
         """
+        if block is self._null_block:
+            return
```

Comment:
Add comment for this new branch ?

Reply:
Hm, do we need to have this? It seems a little weird to have `1` block used even when the system has no requests.